codegen/version.hack

```diff
@@ -1,10 +1,10 @@
 /**
  * This file is generated. Do not modify it manually!
  *
- * @generated SignedSource<<619797a0cd2d7fe25e5069b1f6ae71e9>>
+ * @generated SignedSource<<806e813d8dae345e5700ae04cb296e87>>
  */
 namespace Facebook\HHAST;
 
 const string SCHEMA_VERSION = '2020-01-22-0001';
 
-const int HHVM_VERSION_ID = 44200;
+const int HHVM_VERSION_ID = 404800;
```

Comment:
If you're regenning just so the codegen is lint clean, please regen with 4.42.0 - the docker images can be handy, or if you're on mac, `brew install hhvm-4.42`

Reply:
Fixed